tests/example_app/example/page1/templates/page_1.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+  <br/>
+  <a href="/page-1/a">page 1 A</a>
+  <a href="/page-1/b">page 1 B</a>
+
+  <h2>Currently on page 1</h2>
+
+  <!-- relative static ".static" -->
+  <p>{{ static_url_for('.static', filename='app.css') }}</p>
+
+  <!-- specific static "page_1.static" -->
+  <p>{{ static_url_for('page_1.static', filename='app.css') }}</p>
+
+  <!-- specific static "page_2.static" -->
+  <p>{{ static_url_for('page_2.static', filename='app.css') }}</p>
+
+
+  <link type="text/css" rel="stylesheet"
+        href="{{ static_url_for('.static', filename='app.css') }}">
+
+{% endblock %}
+
```

Comment:
Can you remove this extra new line?

Reply:
[27947c4 improved spacing consistency in jinja files](https://github.com/nickjj/flask-static-digest/pull/33/commits/27947c47ef46de2dbfc486e75705926ae6729b41)